core/src/main/scala/cats/data/WriterT.scala

```diff
@@ -457,10 +457,11 @@ private[data] sealed trait WriterTFoldable[F[_], L] extends Foldable[WriterT[F,
   def foldRight[A, B](fa: WriterT[F, L, A], lb: Eval[B])(f: (A, Eval[B]) => Eval[B]): Eval[B] = fa.foldRight(lb)(f)
 }
 
-private[data] sealed trait WriterTTraverse[F[_], L] extends Traverse[WriterT[F, L, ?]] with WriterTFoldable[F, L] {
+private[data] sealed trait WriterTTraverse[F[_], L] extends Traverse[WriterT[F, L, ?]] with WriterTFoldable[F, L] with WriterTFunctor[F, L] {
 
   override implicit def F0: Traverse[F]
 
+  override def map[A, B](fa: WriterT[F, L, A])(f: A => B): WriterT[F, L, B] = super[WriterTFunctor].map(fa)(f)
```

Comment:
I think we can omit this line if we defined `map` in `WriterTFunctor` as `override def map`.

See for example `EitherTFunctor` and `EitherTMonad` (although `EitherTTraverse` doesn't extends `EitherTFunctor` at the moment and neither does `OptionTTraverse` `OptionTFunctor`).

Reply:
Very true, apologies. I'm fixing it. I rushed it in. I have to stop reading those intellij errors. Do you think I should change also `EitherTTraverse ` and `OptionTTraverse ` accordingly ?

Reply:
I think there are probably more cases than just `EitherTTraverse` and `OptionTTraverse` which can be improved, I would leave those for another PR.